server/server.js

```diff
@@ -49,16 +52,21 @@ app.use(errorHandler);
 const PORT = process.env.PORT || 5000;
 const NODE_ENV = process.env.NODE_ENV;
 
-connectDB();
+await connectDB();
+await scrapeTop10Songs();
+setInterval(() => {
+    clearTop10Songs();
```

Comment:
dont clear before you get new results to replace the old ones. what if the  scrapeTop10Songs function fails?